frontend/src/screens/Intro.tsx

```diff
@@ -143,6 +152,17 @@ function Slide({
 
   return (
     <div className="flex flex-col justify-center items-center h-screen gap-8 p-5">
+      {Icon === SelfCustodyIcon ? (
+        <Icon
+          width="211"
+          height="64"
+          viewBox="0 0 211 64"
```

Comment:
it's weird you need to pass this stuff. Isn't the viewbox correctly set in the svg?

Reply:
We ask for a LucideIcon, and in `createLucideIcon` we can only provide svg's children but not the svg props. I think it's because we can pass them like this (See SelfCustodyIcon.tsx)